# do not treat warnings as errors. because there are warnings.
Index: synergy-1.5.0/CMakeLists.txt
===================================================================
--- synergy-1.5.0.orig/CMakeLists.txt	2014-05-25 00:13:49.972372996 +0200
+++ synergy-1.5.0/CMakeLists.txt	2014-05-25 00:13:49.968372996 +0200
@@ -76,7 +76,7 @@
 
 	# warnings as errors:
 	# we have a problem with people checking in code with warnings.
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
+	#set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Werror")
 
 	# For config.h, detect the libraries, functions, etc.
 	include(CheckIncludeFiles)
